website/project/utils.py

```diff
@@ -45,3 +48,18 @@ def recent_public_registrations(n=10):
             continue
         n -= 1
         yield reg
+
+
+@signals.node_updated.connect
+def update_date_modified_for_users(node):
+    update_date_modified_task(node._id)
+
+
+@run_postcommit(once_per_request=False, celery=True)
+@app.task(max_retries=5, default_retry_delay=60)
+def update_date_modified_task(node_id):
+    node = Node.load(node_id)
+    now = datetime.utcnow()
+    for contrib in node.contributors:
+        contrib.date_modified = now
+        contrib.save()
```

Comment:
Do not save the entire contributor object every time any node the contributor was attached to is updated. You are going to have to just update the one field in the most efficient manner possible. There are other examples of postcommit tasks which should give you an idea of how to do this.